extensions/wikidata/module/langs/translation-en.json

```diff
@@ -3,10 +3,10 @@
     "wikidata-extension/edit-wikidata-schema": "Edit Wikidata schema",
     "wikidata-extension/import-wikidata-schema": "Import schema",
     "wikidata-extension/manage-wikidata-account": "Manage Wikidata account",
-    "wikidata-extension/perform-edits-on-wikidata": "Upload edits to Wikidata",
-    "wikidata-extension/export-to-qs": "Export to QuickStatements",
+    "wikidata-extension/perform-edits-on-wikidata": "Wikidata edits...",
+    "wikidata-extension/export-to-qs": "QuickStatements file...",
     "wikidata-extension/export-schema": "Export schema",
-    "wikidata-extension/export-wikidata-schema": "Export Wikidata schema",
+    "wikidata-extension/export-wikidata-schema": "Wikidata schema to file",
```

Comment:
This label is also used in the Wikidata extension menu, and in that context we do need to keep "Export" in the label because the menu contains all sorts of actions, not just export ones. So I would keep this existing string and add a new one without "export" (and without adding "to file" either, see below).

Reply:
> This label is also used in the Wikidata extension menu, and in that context we do need to keep "Export" in the label because the menu contains all sorts of actions, not just export ones. So I would keep this existing string and add a new one without "export" (and without adding "to file" either, see below).

The menu where « Wikidata schema to file » is located, is called « Export ». So, it's not necessary to repeat it, as you do not do with all the export above that also export (though not the same thing, that why I did put in my proposition this item in a section by itself.